core/txdb/txdb_test.go

```diff
@@ -148,18 +148,12 @@ func TestGetBlock(t *testing.T) {
 		BlockHeader: bc.BlockHeader{
 			Version:           1,
 			Height:            1,
-			PreviousBlockHash: [32]byte{'1', '2', '3'},
+			PreviousBlockHash: bc.Hash{0x3132330000000000, 0, 0, 0},
```

Comment:
I think these would be way more readable as

```go
bc.NewHash([32]byte{'1', '2', '3'})
```

throughout this file in particular.